internal/config/settings.go

```diff
@@ -93,7 +93,7 @@ var defaultCommonSettings = map[string]interface{}{
 	"softwrap":        false,
 	"splitbottom":     true,
 	"splitright":      true,
-	"statusformatl":   "$(filename) $(modified)($(line),$(col)) $(status.paste)| ft:$(opt:filetype) | $(opt:fileformat) | $(opt:encoding)",
+	"statusformatl":   "$(filename) $(modified)($(line),$(col)) $(status.paste)| ft:$(opt:filetype) | $(opt:fileformat) | $(opt:encoding) | $(overwrite)",
```

Comment:
This introduces ` | `  even when `[overwrite]` is not displayed.

Why not put `$(overwrite)` right after `$(modified)`? Especially since `[ro]` and `[overwrite]` are now mutually exclusive, so it seems a good idea to put them at the same place.